sql/6_1_0-to-7_0_0_upgrade.sql

```diff
@@ -666,4 +666,8 @@ SET sql_mode = @currentSQLMode;
 
 #IfMissingColumn insurance_companies cqm_sop
 ALTER TABLE `insurance_companies` ADD `cqm_sop` int DEFAULT NULL COMMENT 'HL7 Source of Payment for eCQMs';
+#EndIf
+
+#IfNotRow2D list_options list_id order_type option_id order
+INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES ('order_type','order','Order',90,0);
 #EndIf
```

Comment:
needs new line at end of file. There are several reason for this one being it helps ensure our upgrade parser get the last item. Most times it may work but, I wouldn't count on it!